cfme/tests/services/test_catalog_item.py

```diff
@@ -91,13 +93,29 @@ def test_catalog_item_crud(appliance, dialog, catalog):
         catalog=catalog,
         dialog=dialog
     )
+
+    # CREATE
     view = cat_item.create_view(AllCatalogItemView)
     assert view.is_displayed
-    view.flash.assert_success_message('Service Catalog Item "{}" was added'.format(cat_item.name))
+    view.flash.assert_success_message('Service Catalog Item "{}" was added'
+                                      .format(cat_item.name))
     assert cat_item.exists
+
+    # EDIT
     with update(cat_item):
         cat_item.description = "my edited description"
+    view.flash.assert_success_message('Service Catalog Item "{}" was saved'
+                                      .format(cat_item.name))
+
+    view = navigate_to(cat_item, 'Edit')
+    view.cancel.click()
+    view = cat_item.create_view(DetailsCatalogItemView)
```

Comment:
`cat_item.create_view(DetailsCatalogItemView, wait="10s")`

Reply:
I'm explicitly asserting against view.wait_displayed after this, so I don't care to have create_view wait.

Reply:
Indirectly, we are doing same :+1: waiting for default "10s"